msteams-platform/bots/bot-features.md

```diff
@@ -48,105 +47,6 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
 
 Webhooks and connectors connect your bot to your web services. Using webhooks and connectors, you can create a bot for basic interaction, such as creating a workflow or other simple commands. They're available only in the team where you create them and are intended for simple processes specific to your company's workflow. For more information, see [what are webhooks and connectors](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md).
```

Comment:
Webhook and connectors are not available in GCCH. Do we need to document that here or we cover it someplace else?

Reply:
This is Mentioned in another location. Sorry for the delay in messages.